Cue/app/actions/login.js

```diff
@@ -26,35 +29,32 @@ function logOut(): Action {
 }
 
 
-
-async function loadUsername(name): PromiseAction {
+async function loadUsername(name: string): PromiseAction {
```

Comment:
Minor nitpick, but since this function isn't actually async, you can remove the `async` declaration and update the return type to just `Action`.